lib/rules/no-unsupported-features.js

```diff
@@ -115,7 +117,10 @@ module.exports = {
       forbiddenVBindPropModifierShorthand:
         '`.prop` shorthand are not supported until Vue.js "3.2.0".',
       forbiddenVBindAttrModifier:
-        '`.attr` modifiers on `v-bind` are not supported until Vue.js "3.2.0".'
+        '`.attr` modifiers on `v-bind` are not supported until Vue.js "3.2.0".',
+      // Vue.js 3.3.0+
+      forbiddenGenericAttribute:
+        '`generic` attribute is not supported until Vue.js "3.3.0".'
```

Comment:
Does it really need to be reported? Adding a `generic` attribute won't do anything on Vue < 3.3, but it also won't hurt.

Reply:
That's a good point! I will close this PR 😅